### PR TITLE
bugfix: fix the inconsistent configuration item names related to LoadBalance

### LINKED ISSUE #5230: inconsistent configuration item names
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
配置名 visualNodes 与 LoadBalanceProperties 类里的field virtualNodes 名不一致，导致在 SpringBootConfigurationProvider 下，获取字段时不存在，导致返回配置值为 null。
也就是说在springboot下，配置了seata.client.load-balance.visual-nodes 或 seata.client.load-balance.virtual-nodes都会导致配置读取不到。
image

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- Seata version:  
- OS :
- Others:

### PR DESCRIPTION
**Pull request by xingfudeshi**
- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did
fix the inconsistent configuration item names related to LoadBalance

### Ⅱ. Does this pull request fix one issue?

fixes #5230 

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
I've provided one.

### CODE REVIEW

File: common/src/main/java/io/seata/common/DefaultValues.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ public interface DefaultValues {
 
     boolean DEFAULT_SERVER_ENABLE_CHECK_AUTH = true;
 
-    String DEFAULT_LOAD_BALANCE = "XID";
```
Comment (a364176773) [inline common/src/main/java/io/seata/common/DefaultValues.java:117]: 默认要使用XID负载均衡，1.5.2改动

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/ClientPropertiesTest.java [hunk 2]
```diff
@@ -108,7 +108,7 @@ public void testUndoProperties() {
 
     @Test
     public void testLoadBalanceProperties() {
-        assertEquals("XID", context.getBean(LoadBalanceProperties.class).getType());
+        assertEquals("RandomLoadBalance", context.getBean(LoadBalanceProperties.class).getType());
```
Comment (a364176773) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/ClientPropertiesTest.java:111]: XID

File: discovery/seata-discovery-core/src/test/resources/file.conf [hunk 3]
```diff
@@ -18,6 +18,6 @@
 client {
     loadBalance {
         type = "RandomLoadBalance"
```
Comment (a364176773) [inline discovery/seata-discovery-core/src/test/resources/file.conf:20]: XID

File: common/src/main/java/io/seata/common/DefaultValues.java (hunk 1 above)
Comment (xingfudeshi) [reply common/src/main/java/io/seata/common/DefaultValues.java]: done

File: discovery/seata-discovery-core/src/test/resources/file.conf (hunk 3 above)
Comment (xingfudeshi) [reply discovery/seata-discovery-core/src/test/resources/file.conf]: done

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/ClientPropertiesTest.java (hunk 2 above)
Comment (xingfudeshi) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/ClientPropertiesTest.java]: done

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 4]
```diff
@@ -348,7 +348,7 @@
       "name": "seata.client.load-balance.type",
       "values": [
         {
-          "value": "RandomLoadBalance",
+          "value": "XID",
```
Comment (a364176773) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/resources/META-INF/additional-spring-configuration-metadata.json:351]: 这里一个应该是加一个XID的方式，而不是直接替换吧？

Comment (xingfudeshi) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/resources/META-INF/additional-spring-configuration-metadata.json:351]: done

**Review by a364176773 — APPROVED** [review]
LGTM

**Review by wangliang181230 — APPROVED** [review]
LGTM

**Comment by wangliang181230** [comment]
develop.md里加下日志吧。

**Comment by xingfudeshi** [comment]
> develop.md里加下日志吧。

done

**Review by slievrly — COMMENTED** [review]
LGTM

**Pull request merged**